tests/pytest/test_regression.py

```diff
@@ -106,6 +116,15 @@ def run_model(rundir, model_image):
         )
 
 
+def run_model_sarus(rundir, model_image):
+    shutil.copy(os.path.join(TEST_DIR, "run_files/job_jenkins_sarus"), os.path.join(rundir, "job_jenkins_sarus"))
+    # run job_jenkins_sarus with env var FV3_CONTAINER set to model_image
+    env = os.environ.copy()
+    env["FV3_CONTAINER"] = model_image
+    env["SCRATCH_DIR"] = rundir
+    subprocess.check_call(["sbatch", "--wait", "job_jenkins_sarus"], env=env, cwd=rundir)
+
+
 def check_md5sum(run_dir, md5sum_filename):
     subprocess.check_call(["md5sum", "-c", md5sum_filename], cwd=run_dir)
```

Comment:
Have you tried running this md5sum test with both base images: ubuntu 19.10 and ubuntu 18.04 (the CUDA 10.2 one)?  I found that I could not get all the output and restart files to agree with the same md5 checksum when running with the Ubuntu 18.04 base image.